app/services/search/search_builder.rb

```diff
@@ -49,7 +48,8 @@ def build_search_replica
   end
 
   def build_suggestions
-    @wider_radius_suggestions = Search::SuggestionsBuilder.new(search_params, @params_hash[:radius]).radius_suggestions
+    return @wider_radius_suggestions = Search::RadiusSuggestionsBuilder.new(search_params, @params_hash[:radius]).radius_suggestions if @point_coordinates.present?
+    @wider_radius_suggestions = Search::BuffersSuggestionsBuilder.new(@params_hash[:location], search_params).buffer_suggestions
```

Comment:
We only want to do this if a polygon search has been carried out

Reply:
I believe this has been changed now take another look and see if you're happy